ddtrace/tracer/option.go

```diff
@@ -627,6 +636,26 @@ func WithLogStartup(enabled bool) StartOption {
 	}
 }
 
+// WithProfilerCodeHotspots enables the code hotspots integration between
+// tracing and profiling. This is done by automatically attaching goroutine
+// labels called "span id" and "local root span id" when new spans are created.
+// You should not use these label names in your own code.
+func WithProfilerCodeHotspots(enabled bool) StartOption {
```

Comment:
```suggestion
func WithProfilerCodeHotspots() StartOption {
```
I might've asked before, sorry if so, I forgot. Why do we need a boolean here? The presence or absence of this option already toggles the functionality. 

Same question for the one below.

Reply:
The question came up before on another PR: https://github.com/DataDog/dd-trace-go/pull/1038#discussion_r740060222

Below is a screenshot because the link above may not correctly open the hidden discussion.

![image](https://user-images.githubusercontent.com/15000/143129932-46858350-066a-4016-aef6-77a3ec79d443.png)

Based on the conclusion in the other PR, I hope a bool is fine here as well.

Reply:
My bad. 🧠 = 🥔

Reply:
No worries, it also took me a while to realize that we had this conversation on another PR, not this one, so I kept looking for it for quite a bit, lol : p.